packages/twenty-server/src/engine/integrations/llm-prompt-template/drivers/file.driver.ts

```diff
@@ -0,0 +1,42 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
+import { PromptTemplate } from '@langchain/core/prompts';
+
+import { LLMPromptTemplateDriver } from 'src/engine/integrations/llm-prompt-template/drivers/interfaces/llm-prompt-template-driver.interface';
+import { LLMPromptTemplateEnvVar } from 'src/engine/integrations/llm-prompt-template/interfaces/llm-prompt-template-name.interface';
+import { PromptTemplateInput } from 'src/engine/integrations/llm-prompt-template/interfaces/llm-prompt-template.interface';
+
+import { resolveAbsolutePath } from 'src/utils/resolve-absolute-path';
+
+export class FileDriver implements LLMPromptTemplateDriver {
+  constructor() {}
+
+  async getPromptTemplate<T extends LLMPromptTemplateEnvVar>(
+    promptTemplateName: string,
+  ): Promise<PromptTemplate<PromptTemplateInput<T>, any>> {
+    const filePath = join(
+      resolveAbsolutePath('.llm-prompt-templates'),
```

Comment:
I understand why you've made this choice and it's a good choice. But subjectively I would prefer to keep the root clean if you just put it in within `/llm-prompt-template/` folder. Maybe create folders within driver (/file/file.driver.ts, /langfuse/langfuse.driver.ts ; and put it in /llm-prompt-template/drivers/file/templates/*.txt). Thanks